values.yaml

```diff
@@ -1,20 +1,28 @@
 name: outline
 image:
   repository: outlinewiki/outline
-  tag: 0.60.3
+  tag: 0.61.1
   pullPolicy: IfNotPresent
 replicas: 1
 env:
-  # SLACK_KEY: "slack key in slack oidc context"
-  # SLACK_SECRET: "slack secret in slack oidc context"
+  # SLACK_KEY: "slack client id"
+  # SLACK_SECRET: "slack client secret"
+  # SMTP_HOST: "mail.yourdomain.tld"
+  # SMTP_PORT: "465"
+  # SMTP_USERNAME: ""
+  # SMTP_PASSWORD: ""
+  # SMTP_FROM_EMAIL: ""
+  # SMTP_REPLY_EMAIL: ""
+  # SMTP_SECURE: "true"
 envSecrets:
-secretKey: "5974e36f82085f057223565ac2d0d01427a90248ade611108b4b937672649bd4" # MUST be replaced! Generate a hex-encoded 32-byte random key. You should use `openssl rand -hex 32`
-utilsSecret: "8c5a14973325ba56694a99856e6be330812b875e5f683dd37022cf76444774bb" # Generate a unique random key. The format is not important but you could still use `openssl rand -hex 32`
+secretKey: "generate_a_new_one" # MUST be replaced! Generate a hex-encoded 32-byte random key. You should use `openssl rand -hex 32`
+utilsSecret: "generate_a_new_one" # Generate a unique random key. The format is not important but you could still use `openssl rand -hex 32`
```

Comment:
There should be only one pattern, defaults or hint secrets. In the next lines in values.yaml instead furthermore defaults are used instead of hint secrets like "generate_a_new_one", so it's mixed pattern. We should avoid this. I'm also a fan of fast deployments for testing cases without the need to provide the keys via setter in such cases. Did you test wether the chart ist totally starting using the hint secrets since often the boot process aborts because of too short keys (not having 32 bytes).

Reply:
No, I did not test it and if it would not work I would find it even better: To give the user working, randomized, default keys is an invitation to use them instead of reading the docs and change them.

But you are right that we should not use mixed patterns, so I changed the other passwords too.